src/main/preload/modalPreload.js

```diff
@@ -6,10 +6,29 @@
 
 import {ipcRenderer} from 'electron';
 
-import {MODAL_CANCEL, MODAL_RESULT, MODAL_INFO, RETRIEVE_MODAL_INFO, MODAL_SEND_IPC_MESSAGE, GET_DARK_MODE, DARK_MODE_CHANGE} from 'common/communication';
+import {
+    MODAL_CANCEL,
+    MODAL_RESULT,
+    MODAL_INFO,
+    RETRIEVE_MODAL_INFO,
+    MODAL_SEND_IPC_MESSAGE,
+    GET_DARK_MODE,
+    DARK_MODE_CHANGE,
+    GET_MODAL_UNCLOSEABLE,
+    MODAL_UNCLOSEABLE,
+} from 'common/communication';
 
 console.log('preloaded for the modal!');
 
+let uncloseable = false;
```

Comment:
Something about storing this state here feels weird to me, but I don't really have a better suggestion of how to do it.

I could see it potentially causing problems if we have multiple modals visible at once since the newest one will dictate whether or not any of the modals are closeable, but I don't think we have any case that this happens currently anyway.

Reply:
This actually won't happen because each instance of this code is in its own tab (or BrowserView), so the instance of `uncloseable` is only related to the specific modal that was opened. It is a weird place to store the state, but I don't think I need to create a ModalWrapper component just for this bit of state at this time.